packages/liveblocks-node/src/__tests__/Session.test.ts

```diff
@@ -95,6 +104,12 @@ describe("authorization (new API)", () => {
     );
   });
 
+  test("allows asterisk as pattern", () => {
+    expect(makeSession().allow("*", [P1]).serializePermissions()).toEqual({
+      "*": [P1],
+    });
```

Comment:
Can't believe we missed this case! 🤦‍♂️ Totally my bad, thanks for catching and fixing this!

Reply:
All good! It wasn't needed before Notifications